crates/stages/src/sets.rs

```diff
@@ -12,15 +12,16 @@
 //! ```no_run
 //! # use reth_stages::Pipeline;
 //! # use reth_stages::sets::{OfflineStages};
-//! # use reth_revm::EvmProcessorFactory;
 //! # use reth_primitives::{PruneModes, MAINNET};
 //! # use reth_evm_ethereum::EthEvmConfig;
 //! # use reth_provider::StaticFileProviderFactory;
 //! # use reth_provider::test_utils::create_test_provider_factory;
 //! # use reth_static_file::StaticFileProducer;
 //! # use reth_config::config::EtlConfig;
+//! # use reth_evm::execute::BlockExecutorProvider;
+//!
+//! fn create(exec: impl BlockExecutorProvider) {
```

Comment:
This example shows up weird, `#` at the beginning of a line only hides that single line.